pkg/oci/containerize_test.go

```diff
@@ -0,0 +1,73 @@
+package oci
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+)
+
+// Test_validatedLinkTaarget ensures that the function disallows
+// links which are absolute or refer to targets outside the given root, in
+// addition to the basic job of returning the value of reading the link.
+func Test_validatedLinkTarget(t *testing.T) {
+	root := "testdata/test-links"
+
+	allRuntimes := []string{} // optional list of runtimes a test applies to
+
+	testAppliesToCurrentRuntime := func(testRuntimes []string) bool {
```

Comment:
Couldn't we just use `slices.Contains()`.

Reply:
Ah yes, I forgot this method exists.  I am so used to Go forcing things be done the long way.